content/docs/user-guide/assets/pipeline/asset-builders.md

```diff
@@ -38,3 +38,58 @@ Process Job generates the product asset and product dependencies. The Asset Buil
 
 Process Job is multithreaded. Several Asset Builders can run multiple process jobs simultaneously.
+
+## Asset Builder Tips and Tricks
+
+### Loading Other Files
+
+Sometimes, when authoring a builder, it may be necessary to load other files besides the primary source file to finish this process. Here are some scenarios that this can come up, and best practices for handling them.
+
+#### Understanding File Locations for Asset Processing
+
+Files relevant to asset processing can exist in three core locations:
+1. The Asset Cache is where all product assets exist.
+1. Scan directories are the only place source assets can exist.
+1. Source files (not assets) can be on any drive in any location.
+
+Scan directories tend to be one of two core locations. There are other locations, but these are the primary ones. Read more about [scan directories here:](/docs/user-guide/assets/pipeline/scan-directories/)
+1. The asset folder for the current project.
+1. The asset folder for each gem enabled for the project.
+
+Individual Gems, the O3DE project in use, and the engine itself can all be installed to different drives.
+
+If you have source assets referencing each other, the safest way to reference one source asset from another is with that asset's UUID, because it will be stable across drives. Read more about [asset identifiers here.](/docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers/#asset-identifiers) Relative paths between source assets may not function as expected, if those source assets are in scan folders across different root directories, there may not be a way to build that reference. Also, different members of a team may install their project, engine, and gems differently than each other, making relative paths in source assets not stable for the team.
+
+The asset cache is more stable, relative paths between two product assets won't change unless the source asset that generating them is moved, or the associated builder is updated to change how it outputs product assets. Even still, it's recommended to have product assets reference each other with asset ID and not relative path, because asset IDs are more stable long term than relative paths.
+
+Product assets should never reference other product assets with absolute path because this will result in the hash of the contents of product assets being unique per machine that generates product assets. When authoring a builder, the complete lifecycle of asset management for a project should be kept in mind. Keeping product assets stable across machines will ensure that the gathering modified assets step of generating a patch for a live game is accurate. You can read more about [asset bundling here.](docs/user-guide/packaging/asset-bundler/overview/)
+
+
+#### Loading Product Assets in Process Job
```

Comment:
As per previous heading suggestion. Only shortened 3 characters, but every little bit helps. If you can come up with a shorter heading, please do.

```suggestion
### Load product assets in Process Job
```